presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -96,14 +98,6 @@
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 import static org.apache.hadoop.hive.serde2.objectinspector.ObjectInspector.Category.PRIMITIVE;
-import static org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName.BINARY;
```

Comment:
any reason remove these constants? were added for schema verification
might trigger compile errors

Reply:
@zhenxiao : Will make sure Travis is green in #13473 before merge :)